packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -446,6 +470,24 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
         context,
         _isChildExpanded(index),
       );
+      final Widget expansionIndicator = child.expansionIndicator != null
```

Comment:
I think this block of code is complex enough that ternary operators (widget = bool ? val1 : val2)  shouldn't be used 
as it gets pretty hard to read. Maybe use an if/else statement here instead.

Reply:
the ternary operator  was replaced by if-else, but 'expansionIndicator' cannot be named 'final' either;